### DISCUSSION TITLE
Managing shared libraries

### DISCUSSION THREAD

**Discussion by pulsejet** [nextcloud/recognize#751] [General]
With a recent update, recognize duplicates detections and clusters for each user a file is shared with. I'm wondering how well this works with large shared libraries that need significant curation. For example, if Alice shares a folder with Bob and then does some (a lot of) curation (such as moving around photos between clusters, deleting clusters etc.) then Bob would have to do everything again. Maybe there needs to be a way to sync these clusters and detections between users.

**Comment by vitis586** [comment]
Does the Face Recognition work the same way? I have not tried it yet.

**Reply by pulsejet** [reply]
IIRC face recognition still uses the old behavior, i.e. the owner of the files owns the faces. The upside is that the changes propagate to everyone. Downside is that if the other user has some files of their own with the same face, there will be two separate clusters in the UI. This could, of course, be handled in the UI itself by collapsing duplicate clusters, but managing such a cross owned library is another tricky matter.

**Comment by Croydon** [comment]
See also #948
